### PR TITLE
Allow multiple multipliers.

### PR DESCRIPTION
**Pull request by lasiotus**
Currently, only two multipliers are allowed. But it is common for a lisp program to have (* N1 N2 N3 N4) expressions. Moreover, (+ N1 N2 N3 N4) expressions are already supported.

### CODE REVIEW

File: src/default_environment.rs [hunk 1]
```diff
@@ -395,26 +395,54 @@ pub fn default_env() -> Env {
...
+                                }
+                            }
+                        }
+                        Value::Float(t) => Value::Float(t * *x),
+                        _ => unreachable!("Will only ever assign int/float to `product`"),
+                    },
+                    _ => {
+                        return Err(RuntimeError {
+                            msg: format!(
+                                "Function \"+\" requires arguments to be numbers; found {}",
```
Comment (brundonsmith) [inline src/default_environment.rs:437]: `+` -> `*`

**Comment by brundonsmith** [comment]
Two comments, otherwise this looks good 👍

File: src/default_environment.rs (hunk 1 above)
Comment (lasiotus) [reply src/default_environment.rs]: Done.

**Comment by lasiotus** [comment]
Hmm... Found only one comment, fixed. :)

**Comment by brundonsmith** [comment]
Other one was this:
Screenshot 2022-12-20 at 11 40 11 PM

**Pull request merged**